### ISSUE TITLE
Quick stack does not work for some cases.

### ISSUE THREAD

**Issue by sors89** [Soof4/QoL#8]
The disable quick stack feature does not seem to work when you are opening a chest then click the quick stack button, happens both mobile and pc

**Comment by LiteralSoofa** [comment]
Yup i know this bug but won't bother to fix it because the reason i added this option is to stop people from quick stacking their items to protected chests and since players won't be able to open protected chests there is no need to put effort into fixing.

**Comment by sors89** [comment]
> Yup i know this bug but won't bother to fix it because the reason i added this option is to stop people from quick stacking their items to protected chests and since players won't be able to open protected chests there is no need to put effort into fixing.

Oh, i thought this feature is to prevent players from duping items. More [details](https://forums.terraria.org/index.php?threads/multiplayer-duplication-bug.87830/)

**Comment by Hexagonforce12** [comment]
> Yup i know this bug but won't bother to fix it because the reason i added this option is to stop people from quick stacking their items to protected chests and since players won't be able to open protected chests there is no need to put effort into fixing.

Maybe create a system so that items won't go into chests in the region, if use Quickstack

**Comment by LiteralSoofa** [comment]
that sounds very complicated 😓
